common/src/main/java/io/netty/util/concurrent/AbstractFuture.java

```diff
@@ -53,6 +54,6 @@ public V get(long timeout, TimeUnit unit) throws InterruptedException, Execution
             }
             throw new ExecutionException(cause);
         }
-        throw new TimeoutException();
+        throw new TimeoutException("timeout after " + timeout + " " + unit.name().toLowerCase(Locale.ENGLISH));
```

Comment:
Alternatively, I can convert `timeout` to ms if you'd like to avoid `toLowerCase`